### PR TITLE
Create a popup  instead of category create create update and delate pages

### PR DESCRIPTION
**Pull request by Otabek7667a**
change to popup

### CODE REVIEW

File: ExpenseTracker/ExpenseTracker/appsettings.json [hunk 1]
```diff
@@ -5,5 +5,18 @@
...
+    "From": "otabekxudoynazarov131@gmail.com",
+    "SmtpServer": "smtp.gmail.com",
+    "Port": 465,
+    "Username": "otabekxudoynazarov131@gmail.com",
+    "Password": "sixq trwl zebo xfza"
+  },
+
+  "SyncfusionKey": "Ngo9BigBOggjHTQxAR8/V1NCaF5cXmZCf1FpRmJGdld5fUVHYVZUTXxaS00DNHVRdkdnWXdedHRQRGRZWEV+WkQ="
 }
+
```
Comment (Ramadan1011) [inline ExpenseTracker/ExpenseTracker/appsettings.json:22]: You shouldn't push appsettings.json file

**Comment by Otabek7667a** [comment]
Ok tushinarli qanday bekor qilsam bo'ladi

Chor, 2-okt, 2024, 14:35 Ramadan1011 ***@***.***>:

**Comment by Ramadan1011** [comment]
You may remove pull request

**Comment by Otabek7667a** [comment]
Done ok

Chor, 2-okt, 2024, 15:52 Ramadan1011 ***@***.***>:

**Pull request closed without merging**